src/localization/en.js

```diff
@@ -262,6 +262,8 @@ export default {
       tabTitleStackedIncomes:  `Stacked Incomes`,
       tabTitleBenefitPrograms: `Benefit Programs`,
       chartsHeader:            `With the new pay, how could your benefits change?`,
+      warningMessage:          `This tool is in testing and these numbers might not be right. If they're not, we'd appreciate your feedback.`,
```

Comment:
Since these are new ones, it may be worth just starting them out with version numbers instead of going through this intermediate step. In fact, maybe it's worth versioning all the snippets that you're touching here. Would you be up for that?

Reply:
👍 Yep I think it makes sense to change all the snippet names I'm touching since this is the direction we're going. Just pushed an update to that effect.